### ISSUE TITLE
BigBlueButton v2.2 Presentation suddenly not working on all servers

### ISSUE THREAD

**Issue by ali-rahimi-coder** [bigbluebutton/bigbluebutton#13373]
All of a sudden today, I noticed 3 different BigBlueButton v2.2 instances on 3 different servers with 3 different providers don't show the presentation module. I can upload presentations, but nothing is shown other than a toolbar(just like the image below). I haven't changed anything in the past 2 weeks or so. Restarting using `bbb-conf --restart` doesn't fix the issue, neither does rebooting the servers.

![Screenshot from 2021-10-02 09-09-59](https://user-images.githubusercontent.com/33478509/135705180-b226ac41-be52-4e36-88e6-21af7d00bf61.png)

By the way I don't know if the system was working in October 1st, can the problem be somehow connected to the LetsEncrypt root CA certificate issue of September 30th?!

I should mention that another instance of bigbluebutton running v2.3 is working fine, so this is specific to v2.2.x.(The client asked for v2.2 instead of 2.3, not my choice)

`bbb-conf --check` shows nothing:

```
BigBlueButton Server 2.2.36 (2416)
                    Kernel version: 4.15.0-112-generic
                      Distribution: Ubuntu 16.04.7 LTS (64-bit)
                            Memory: 32935 MB
                         CPU cores: 32

/usr/share/bbb-web/WEB-INF/classes/bigbluebutton.properties (bbb-web)
       bigbluebutton.web.serverURL: https://vc.nfpi.ir
                defaultGuestPolicy: ALWAYS_ACCEPT
                 svgImagesRequired: true

/etc/nginx/sites-available/bigbluebutton (nginx)
                       server_name: vc.nfpi.ir
                              port: 80, [::]:80
                              port: 443 ssl

/opt/freeswitch/etc/freeswitch/vars.xml (FreeSWITCH)
                       local_ip_v4: 86.104.39.246
                   external_rtp_ip: 86.104.39.246
                   external_sip_ip: 86.104.39.246

/opt/freeswitch/etc/freeswitch/sip_profiles/external.xml (FreeSWITCH)
                        ext-rtp-ip: $${local_ip_v4}
                        ext-sip-ip: $${local_ip_v4}
                        ws-binding: 86.104.39.246:5066
                       wss-binding: 86.104.39.246:7443

/usr/local/bigbluebutton/core/scripts/bigbluebutton.yml (record and playback)
                     playback_host: vc.nfpi.ir
                 playback_protocol: https
                            ffmpeg: 4.2.4-1ubuntu0.1bbb1~16.04.1

/etc/bigbluebutton/nginx/sip.nginx (sip.nginx)
                        proxy_pass: 86.104.39.246
                          protocol: http

/usr/local/bigbluebutton/bbb-webrtc-sfu/config/default.yml (Kurento SFU)
                        kurento.ip: 86.104.39.246
                       kurento.url: ws://127.0.0.1:8888/kurento
                    kurento.sip_ip: 86.104.39.246
                    localIpAddress: 86.104.39.246
               recordScreenSharing: true
                     recordWebcams: true
                  codec_video_main: VP8
               codec_video_content: VP8

/usr/share/meteor/bundle/programs/server/assets/app/config/settings.yml (HTML5 client)
                             build: 1157
                        kurentoUrl: wss://vc.nfpi.ir/bbb-webrtc-sfu
                  enableListenOnly: true
                    sipjsHackViaWs: true

/usr/share/bbb-web/WEB-INF/classes/spring/turn-stun-servers.xml (STUN Server)
                              stun:stun.l.google.com:19302


# Potential problems described below
....................
```

**Comment by m-nik** [comment]
Hi, this problem originated from the expired CA that used by Let's Encrypt, ISRG Root X1 on 30 Dec 2021, if you're using Ubuntu 16.04.

**Comment by ali-rahimi-coder** [comment]
> Hi, this problem originated from the expired CA that used by Let's Encrypt, ISRG Root X1 on 30 Dec 2021, if you're using Ubuntu 16.04.

Thanks for the reply. Do you know of any way to fix this except upgrading to Ubuntu 18.04 and BBB 2.3?

**Comment by ali-rahimi-coder** [comment]
So after checking the error log of html5 I found this answer which worked like a charm on my 3 bigbluebutton 2.2 installations on Ubuntu 16.04

https://github.com/bigbluebutton/bigbluebutton/issues/6139#issuecomment-932679845
